src/main/java/core/basesyntax/service/impl/WriterServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.FruitTransaction;
+import core.basesyntax.service.WriterService;
+import java.io.FileWriter;
+import java.util.List;
+
+public class WriterServiceImpl implements WriterService {
+    @Override
+    public void writeToCsv(List<FruitTransaction> list, String filePath) {
+        try {
+            FileWriter fileWriter = new FileWriter(filePath);
+            for (FruitTransaction fruitTransaction : list) {
+                fileWriter.write(fruitTransaction
+                        .getFruit()
+                        + ","
+                        + fruitTransaction.getQuantity() + "\n");
+            }
+            System.out.println("Successfully written");
```

Comment:
Avoid using System.out.println for logging or user notifications in service classes. Consider using a logging framework instead.